druid/src/scroll_component.rs

```diff
@@ -515,13 +516,63 @@ impl ScrollComponent {
     }
 }
 
+/// The default handling of the [`SCROLL_TO_VIEW`] notification for a scrolling container.
+///
+/// The [`SCROLL_TO_VIEW`] notification is send when [`scroll_to_view`] or [`scroll_area_to_view`]
+/// are called.
+///
+/// [`SCROLL_TO_VIEW`]: crate::commands::SCROLL_TO_VIEW
+/// [`scroll_to_view`]: crate::EventCtx::scroll_to_view()
+/// [`scroll_area_to_view`]: crate::EventCtx::scroll_area_to_view()
+pub fn default_scroll_to_view_handling(
+    ctx: &mut EventCtx,
+    port: &mut Viewport,
+    global_highlight_rect: Rect,
+) {
+    let global_content_offset = ctx.window_origin().to_vec2() - port.view_origin.to_vec2();
```

Comment:
probably not a good idea to use widget's origin here

Reply:
What is the problem with using the widget's origin here?